### DISCUSSION TITLE
Updating a modified Renovate branch

### DISCUSSION THREAD

**Discussion by irrandon** [renovatebot/renovate#28290] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab self-hosted Renovate: mend/renovate-ce:6.11.0-full

### Please tell us more about your question or problem

Hello,

Is it possible to configure Renovate to continue updating a modified Renovate branch?

Our use case is as follows:

We have a dependency that receives a new version every night. 
When the pipeline fails for the new version, our developers make changes directly to the Renovate branch to adjust to the new version. This can take some time. 
During this time, we still want Renovate to update the dependency in this branch with a new commit.

Thank you

**Comment by irrandon** [comment]
The versions that may be potentially broken are the major and minor releases. The "Nightly" versions consist of patch updates.

**Comment by rarkins** [comment, marked as answer]
If you tick the rebase/retry checkbox in the MR then Renovate will update the branch (force pushing over any changes). It's not possible to update the branch _and_ keep your modifications

**Reply by irrandon** [reply]
Is it possible to integrate this functionality into Renovate as a new feature? (:

**Reply by rarkins** [reply]
No

**Reply by irrandon** [reply]
hm ok thanks for your time (:
